### PR TITLE
Simplify highpass and lowpass filter logging

### PR DESCRIPTION
**Pull request by kcgen**
PVS Studio flagged this in the LPT DAC PR (but not sure why it didn't catch it before)

### CODE REVIEW

File: src/hardware/mixer.cpp [hunk 1]
```diff
@@ -591,67 +591,44 @@ void MixerChannel::AddSilence()
...
-	safe_sprintf(buf, " (%d dB/oct at %d Hz)", order * 6, cutoff_freq);
-	return std::string(buf);
+	assert(state != FilterState::Off);      // we expect only enabled states
+	assert(enum_count<FilterState>() == 3); // catch enum additions/removals
+
+	LOG_MSG("MIXER: %s channel %s filter enabled%s (%d dB/oct at %u Hz)",
+	        channel_name.c_str(),
+	        filter_name.data(),
+	        state == FilterState::On ? "" : " (forced)",
+	        order * 6,
```
Comment (johnnovak) [inline src/hardware/mixer.cpp:606]: Perhaps extract to `db_per_order`?

**Review by johnnovak — APPROVED** [review]
Nice cleanup @kcgen!

File: src/hardware/mixer.cpp (hunk 1 above)
Comment (kcgen) [reply src/hardware/mixer.cpp:606]: Will do!

**Comment by kcgen** [comment]
Thanks for the review, @johnnovak :+1:

**Pull request merged**